qcodes/instrument_drivers/Keysight/keysightb1500/KeysightB1500_base.py

```diff
@@ -43,6 +47,66 @@ def __init__(self, name, address, **kwargs):
             Set the function to OFF in cases that the measurement speed is 
             more important than the measurement accuracy. This roughly halves
             the integration time."""))
+
+        self.add_parameter('use_nplc_for_high_speed_adc',
+                           set_cmd=self._set_nplc_for_high_speed_adc,
+                           get_cmd=self._get_nplc_for_high_speed_adc,
+                           vals=vals.Numbers(0, 1023),
```

Comment:
This does not seem to match the value in the docstring. Should it not be an Ints(1,100) ? validator

Reply:
![Screenshot 2020-05-18 at 11 47 05](https://user-images.githubusercontent.com/18750964/82199002-66f98d80-98fd-11ea-8c19-59307fbc4240.png)
High Speed type =  0 
High speed resolution = 2

Reply:
Just for completeness I added this figure in this PR.  Thanks for catching this, the validator is correct, I will fix the docstring.